test/test_reductions.py

```diff
@@ -499,17 +499,17 @@ def test_all_any_empty(self, device):
         self.assertTrue(x.all())
         self.assertFalse(x.any())
 
-    @dtypesIfCUDA(torch.half, torch.float, torch.double)
-    @dtypes(torch.float, torch.double)
+    @dtypesIfCUDA(torch.half, torch.bfloat16, torch.float, torch.double)
```

Comment:
Is it necessary to test on all these types given OpInfo tests for supported dtypes already? Do we expect different kernels to run for these types?

Reply:
@heitorschueroff, these tests seem to be necessary because `amax` & `amin` `OpInfo` tests don't test for functionality.

As mruberry [explained earlier](https://github.com/pytorch/pytorch/pull/51244#issuecomment-770517153),

> OpInfos are really helpful metadata that let the test suite generate a variety of tests, but they don't tell the test suite whether the function is working as expected or not (this is not entirely true, but for amin and amax we can assume it's the case). So there are also tests for amin and amax functionality in test_reductions.py